docs/serving/openai_compatible_server.md

```diff
@@ -49,7 +49,7 @@ We currently support the following OpenAI APIs:
     - *Note: `suffix` parameter is not supported.*
 - [Chat Completions API](#chat-api) (`/v1/chat/completions`)
     - Only applicable to [text generation models](../models/generative_models.md) with a [chat template](../serving/openai_compatible_server.md#chat-template).
-    - *Note: `parallel_tool_calls` and `user` parameters are ignored.*
```

Comment:
I think it’s worth clarifying that this only implements the semantics for `parallel_tool_calls = False`, and not for `True`, since some models don’t natively support `parallel_tool_calls`.


@bbrowning WDYT?